src/Php70/Resources/stubs/SessionUpdateTimestampHandlerInterface.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+interface SessionUpdateTimestampHandlerInterface
```

Comment:
I see it somehow similar: https://github.com/symfony/polyfill/issues/101#issuecomment-328663214
but probably only things that were not available in 5.x are using this interface, so it shall be safe.
correct?

Reply:
it's safe, this interface is for userland, it's not implemented by core classes.